libr/include/r_userconf.h.acr

```diff
@@ -79,6 +79,7 @@
 #define R2_HOME_CACHEDIR    R_JOIN_2_PATHS (".cache", "radare2")
 
 #define R2_HOME_THEMES      R_JOIN_2_PATHS (R2_HOME_DATADIR, "cons")
+#define R2_HOME_FORTUNES    R_JOIN_2_PATHS (R2_HOME_DATADIR, "fortunes")
```

Comment:
that wont be available for the meson builds i think

Reply:
why